services/github/schemas/v1/webhook-check-run-payload.yml

```diff
@@ -0,0 +1,111 @@
+$schema: "/schemas/common/metaschema.json#"
+title: "GitHub Check Run Webhook Payload"
+description: |
+  Validates GitHub check run event webhook payload.
+
+type: object
+properties:
+  action:
+    type: string
+    enum: ["created", "completed", "rerequested", "requested_action"]
+  check_run:
+    type: object
+    properties:
+      id:
+        type: integer
+        minimum: 1
+      head_sha:
+        type: string
+        pattern: "^[a-f0-9]{40}$"
+      status:
+        type: string
+        enum: ["queued", "in_progress", "completed"]
```

Comment:
github docs list 4 options here: https://docs.github.com/en/webhooks/webhook-events-and-payloads#check_run